parcels/scripts/get_examples.py

```diff
@@ -128,7 +128,7 @@ def main(target_path=None):
         args = parser.parse_args()
         target_path = args.target_path
 
-    if os.path.exists(target_path):
+    if os.path.exists(os.path.join(target_path, "MovingEddies_data")):
```

Comment:
Why only check for one of the paths? is that intentional?

Reply:
what was before checked if the given head path exists. Hence, on my machine and on GEMINI, I was not able to say `parcels_get_examples examples/` because `examples` already existed as a folder. This way, we only check if the actual example data (e.g. the folder of *MovingEddies_data*) have already been pushed into the folder.

Yes, we can be more elaborate, checking for all example data, and then pick up aborted downloads. That seemed in this case just a bit too much - I just wanted to be able to make the tests run the way they are descriped by the linux pipeline.

Reply:
No, is fine like this then